test/kubemark/start-kubemark.sh

```diff
@@ -167,6 +139,35 @@ EOF
   echo "Kubeconfig file for kubemark master written to ${LOCAL_KUBECONFIG}."
 }
 
+# Copy all the necessary resource files (scripts/configs/manifests) to the master.
```

Comment:
Just to follow the convention. It makes it easy to read the functions in order, without having to scroll down to find it.